transformation_test/testdata/logging_processor-tomcat-system/input.log

```diff
@@ -1,22 +1,35 @@
+21-Jul-2025 01:02:43.124 INFO [main] org.apache.catalina.startup.VersionLoggerListener Server number:  9.0.82.0
+21-Jul-2025 01:02:43.124 INFO [main] org.apache.catalina.startup.VersionLoggerListener OS Name:        Linux
+21-Jul-2025 01:02:43.124 INFO [main] org.apache.catalina.startup.VersionLoggerListener OS Version:     5.15.0-86-generic
+21-Jul-2025 01:02:43.124 INFO [main] org.apache.catalina.startup.VersionLoggerListener Architecture:   amd64
+21-Jul-2025 01:02:43.125 INFO [main] org.apache.catalina.startup.VersionLoggerListener Java Home:      /usr/lib/jvm/java-17-openjdk
+21-Jul-2025 01:02:43.125 INFO [main] org.apache.catalina.startup.VersionLoggerListener JVM Version:    17.0.8+7
+21-Jul-2025 01:02:43.125 INFO [main] org.apache.catalina.startup.VersionLoggerListener JVM Vendor:     Eclipse Adoptium
+21-Jul-2025 01:02:43.221 INFO [main] org.apache.coyote.AbstractProtocol Initializing ProtocolHandler ["http-nio-8080"]
+21-Jul-2025 01:02:43.250 INFO [main] org.apache.catalina.core.StandardService Starting service [Catalina]
+21-Jul-2025 01:02:43.250 INFO [main] org.apache.catalina.core.StandardEngine Starting Servlet engine: [Apache Tomcat/9.0.82]
+21-Jul-2025 01:02:43.262 INFO [main] org.apache.catalina.core.AprLifecycleListener Loaded APR based Apache Tomcat Native library [1.2.32] using APR version [1.7.4].
+21-Jul-2025 01:02:43.330 INFO [main] org.apache.catalina.loader.WebappLoader Deploying web application directory [/var/lib/tomcat9/webapps/ROOT]
+21-Jul-2025 01:02:43.456 INFO [main] org.apache.catalina.core.ApplicationContext.log No Spring WebApplicationInitializer types detected on classpath
+21-Jul-2025 01:02:43.555 INFO [main] org.apache.catalina.startup.HostConfig Deployment of web application directory [/var/lib/tomcat9/webapps/ROOT] has finished in [225] ms
+21-Jul-2025 01:02:43.565 INFO [main] org.apache.coyote.AbstractProtocol Starting ProtocolHandler ["http-nio-8080"]
+21-Jul-2025 01:02:43.565 INFO [main] org.apache.catalina.startup.Catalina Server startup in 356 ms
+21-Jul-2025 10:33:41.123 WARN [http-nio-8080-exec-2] org.apache.catalina.authenticator.AuthenticatorBase Security checking request failed for /admin/
+21-Jul-2025 10:33:44.234 ERROR [http-nio-8080-exec-4] org.apache.catalina.core.StandardWrapperValve Servlet.service() for servlet [dispatcher] threw exception
+21-Jul-2025 14:40:01.456 INFO [ContainerBackgroundProcessor[StandardEngine[Catalina]]] org.apache.catalina.core.StandardContext Reloading Context [/myapp] has started
+21-Jul-2025 14:40:01.789 INFO [ContainerBackgroundProcessor[StandardEngine[Catalina]]] org.apache.catalina.core.StandardContext Reloading Context [/myapp] has completed
+21-Jul-2025 16:12:01.567 INFO [http-nio-8080-exec-3] org.apache.catalina.session.ManagerBase Session timeout set to 30 minutes
+21-Jul-2025 17:03:21.101 ERROR [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve Servlet.service() for servlet [dispatcher] threw exception
```

Comment:
Nit : Is it possible to add one last "log" line after the "multiline" example ? This will make sure the "multiline" example is completed.

Now i'm realizing that `MultilineRule` works as a state machine and it keeps looking for the next `cont` line different until it doesn't find it, so there is no ending "state" for this log.

https://github.com/GoogleCloudPlatform/ops-agent/blob/cda38309450bf45aeffc5706fcf60adf179604ff/apps/tomcat.go#L93-L104

Reply:
Thank you for the update! I don't think this is a "bug" per-se, since that is how `Fluent-bit` multiline parsing works . Otel Logging multiline parsing may work a bit differently when we implement it.

After this update, now this is PR is facing the issue that is going to be fixed in https://github.com/GoogleCloudPlatform/ops-agent/pull/2010 . Will merge this after the fix.